consensus-types/types/validator.go

```diff
@@ -365,5 +365,5 @@ func (v *Validator) Status(currentEpoch math.Epoch) (string, error) {
 		}
 		return constants.ValidatorStatusWithdrawalDone, nil
 	}
-	return "", errors.New("invalid validator status")
+	return "", errors.Wrapf(errors.New("invalid validator status"), "validator pubkey: %s, index: %d,", v.Pubkey[:], index)
```

Comment:
you don't need to do this here. You can return the error and wrap it in the called